website/docs/theme-classic.md

```diff
@@ -262,6 +262,28 @@ module.exports = {
 };
 ```
 
+### Navbar doc link
+
+If you want to link to a specific doc, this special navbar item type will render the link to the doc of the provided docId. It will get the class `activeSidebarClassName` as long as you browse a doc of the same sidebar.
```

Comment:
```suggestion
If you want to link to a specific doc, this special navbar item type will render the link to the doc of the provided `docId`. It will get the class `navbar__link--active` as long as you browse a doc of the same sidebar.
```